.pre-commit-config.yaml

```diff
@@ -9,6 +9,7 @@ repos:
       - id: check-toml
       - id: end-of-file-fixer
       - id: trailing-whitespace
+        exclude: ^tests/
```

Comment:
Why are we excluding the entire tests directory?

Reply:
This was forgotten, I meant to exclude tests/data because the tsv output has trailing whitespaces that pre-commit wanted to remove, making the test fail.
These tests are anyway unfortunately skipped at the moment, I kept them as skipped because I still want to use them, probably with less strict assertions. I'm still thinking about the best way to loosen up these tests so they pass.